node/src/accountant/mod.rs

```diff
@@ -1682,8 +1689,8 @@ mod tests {
         );
         assert_eq!(blockchain_bridge_recording.len(), 1);
         assert_using_the_same_logger(&logger_clone, test_name, None)
-        // adjust_payments() did not need a prepared result, which means it wasn't reached
-        // because otherwise this test would've panicked
+        // adjust_payments() did not need any prepared result, which means it couldn't have been
+        // reached because otherwise this test would've panicked
```

Comment:
To make it more clear, can you use this:

```
The adjust_payments() function doesn't require prepared results, indicating it shouldn't have been reached during the test, or it would have caused a panic.
```